osafw-app/App_Code/fw/FwReports.cs

```diff
@@ -22,8 +22,24 @@
 // Utils.cleanupTmpFiles();
 
 
+using Microsoft.Extensions.Options;
 using System;
 using System.Collections;
+using System.Collections.Generic;
+using static System.Net.Mime.MediaTypeNames;
+using System.IO;
+using System.Reflection.PortableExecutable;
+using DocumentFormat.OpenXml;
+using DocumentFormat.OpenXml.Packaging;
+using DocumentFormat.OpenXml.Spreadsheet;
+using DocumentFormat.OpenXml.Wordprocessing;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+using DocumentFormat.OpenXml.Drawing;
+using DocumentFormat.OpenXml.VariantTypes;
+using System.Linq;
+using DocumentFormat.OpenXml.Drawing.Charts;
+using Microsoft.AspNetCore.Http;
```

Comment:
I don't want all this stuff here. FwReports should not know about these. Do everything in `ConvUtils`